modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -0,0 +1,279 @@
+* `Microsoft.Network/routeTables/write`
+* `Microsoft.Network/virtualNetworks/peer/action`
+* `Microsoft.Network/virtualNetworks/virtualNetworkPeerings/read`
+* `Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write`
+====
+
+The following permissions are required for deleting an {product-title} cluster on Microsoft Azure. You can use the same permissions to delete a private {product-title} cluster on Azure.
+
+.Required permissions for deleting authorization resources
+[%collapsible]
+====
+* `Microsoft.Authorization/roleAssignments/delete`
+====
+
+.Required permissions for deleting compute resources
+[%collapsible]
+====
+* `Microsoft.Compute/disks/delete`
+* `Microsoft.Compute/galleries/delete`
+* `Microsoft.Compute/galleries/images/delete`
+* `Microsoft.Compute/galleries/images/versions/delete`
+* `Microsoft.Compute/virtualMachines/delete`
+====
+
+.Required permissions for deleting identity management resources
+[%collapsible]
+====
+* `Microsoft.ManagedIdentity/userAssignedIdentities/delete`
+====
+
+.Required permissions for deleting network resources
+[%collapsible]
+====
+* `Microsoft.Network/dnszones/read`
+* `Microsoft.Network/dnsZones/A/read`
+* `Microsoft.Network/dnsZones/A/delete`
+* `Microsoft.Network/dnsZones/CNAME/read`
+* `Microsoft.Network/dnsZones/CNAME/delete`
+* `Microsoft.Network/loadBalancers/delete`
+* `Microsoft.Network/networkInterfaces/delete`
+* `Microsoft.Network/networkSecurityGroups/delete`
+* `Microsoft.Network/privateDnsZones/read`
+* `Microsoft.Network/privateDnsZones/A/read`
+* `Microsoft.Network/privateDnsZones/delete`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/delete`
+* `Microsoft.Network/publicIPAddresses/delete`
+* `Microsoft.Network/virtualNetworks/delete`
+====
+[NOTE]
+====
+The following permissions are not required to delete a private {product-title} cluster on Azure.
+
+* `Microsoft.Network/dnszones/read`
+* `Microsoft.Network/dnsZones/A/read`
+* `Microsoft.Network/dnsZones/A/delete`
+* `Microsoft.Network/dnsZones/CNAME/read`
+* `Microsoft.Network/dnsZones/CNAME/delete`
+====
+
+.Required permissions for checking the health of resources
+[%collapsible]
+====
+* `Microsoft.Resourcehealth/healthevent/Activated/action`
+* `Microsoft.Resourcehealth/healthevent/Resolved/action`
+* `Microsoft.Resourcehealth/healthevent/Updated/action`
+====
+
+.Required permissions for deleting resource group
+[%collapsible]
+====
+* `Microsoft.Resources/subscriptions/resourcegroups/delete`
+====
+
+.Required permissions for deleting storage resources
+[%collapsible]
+====
+* `Microsoft.Storage/storageAccounts/delete`
+* `Microsoft.Storage/storageAccounts/listKeys/action`
+====
+
+[NOTE]
+====
+You can create a custom role with all the permissions and assign that custom role to the service principal. To create a custom role in the Azure portal, see link:https://learn.microsoft.com/en-us/azure/role-based-access-control/custom-roles[Azure custom roles] in the Azure documentation.
+====
```

Comment:
I believe this note is no longer needed since you added the information to the start of this module.

Reply:
Addressed